### PR TITLE
add django-rest-swagger to the requirements.txt

### PR DESCRIPTION
**Pull request by mxmoss**

### CODE REVIEW

**Comment by mxmoss** [comment]
Changed over to use PostGres database
Instead of breaking the endpoints that serve up the CSV data, I added the postgres endpoints as new ones. Eventually, however, we'll want to have a single endpoint for ORCB and one endpoint for KPM.

**Comment by jimtyhurst** [comment]
I tested the recent additions to this branch, checking that multiple filters worked for the `/ocrb-prod` endpoint.

I would have left this as three separate pull requests, because they are independent of each other and could be reviewed and tested more easily separately:

- the original "add django-rest-swagger dependency" pull request, which fixed a defect.
- add PostgreSQL configuration, which duplicates pull request #35 .
- refactor `/ocrb` and `/kpm` endpoints to use the database, rather than reading data from the CSV file.

However, now that you have all this working together, I think you should merge it. _However_, please merge the current state of `master` into your branch `moss-feat`, so that you integrate with recent changes before doing the merge back into `master`.

File: budget_proj/budget_app/urls.py [hunk 1]
```diff
@@ -9,5 +9,7 @@
 	url(r'^$', schema_view),
     url(r'^kpm/$', views.ListKpm.as_view()),
     url(r'^ocrb/$', views.ListOcrb.as_view()),
+    url(r'^ocrb-prod/$', views.OcrbList.as_view()),
```
Comment (jimtyhurst) [inline budget_proj/budget_app/urls.py]: Now that we have experimented with three ways of getting OCRB data (`/ocrb`, `/ocrb-prod`, and `/summary`) , we should spend a little more time on the API design at the team meeting next week to reconcile these three into a single endpoint.

File: budget_proj/budget_app/views.py [hunk 2]
```diff
@@ -60,6 +60,27 @@ class gives you a convenient way to declare views quickly when you only
...
+    queryset = models.OCRB.objects.all()
+    serializer_class = serializers.OcrbSerializer
+    def get(self, request, *args, **kwargs):
+        #check for query params
+        #eg: http://127.0.0.1:8000/ocrb-prod/?amount=0
+        k = request.GET.keys()
+        filter_dict = {}
+        if(k):
+            for key, value in request.GET.items():
+                filter_dict[key] = value
```
Comment (jimtyhurst) [inline budget_proj/budget_app/views.py]: I like your use of a dictionary of query parameters better than the long list of if-statements that I implemented in the `FindOperatingAndCapitalRequirements` class.

**Pull request merged**